### PR TITLE
refactor: opt out marketing related emails

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-531 #done

### CODE REVIEW

File: packages/shared/src/components/auth/SocialRegistrationForm.tsx [hunk 1]
```diff
@@ -38,6 +39,11 @@ export type SocialRegistrationFormValues = Omit<
   'method' | 'provider'
 >;
 
+export interface SocialRegistrationFormParameters
+  extends SocialRegistrationFormValues {
```
Comment (rebelchris) [inline packages/shared/src/components/auth/SocialRegistrationForm.tsx:43]: Why can't we add them to the initial socialRegistrationFormValues?

File: packages/shared/src/components/auth/RegistrationForm.tsx [hunk 2]
```diff
@@ -31,6 +32,10 @@ export type RegistrationFormValues = Omit<
   'method' | 'provider'
 >;
 
+interface EmailRegistrationForm extends RegistrationFormValues {
```
Comment (rebelchris) [inline packages/shared/src/components/auth/RegistrationForm.tsx:35]: Same here, can we not simply add it to the main one?

**Review by rebelchris — CHANGES_REQUESTED** [review]
Just a small comment

File: packages/shared/src/components/auth/RegistrationForm.tsx (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/components/auth/RegistrationForm.tsx:35]: I don't know why I did this. Lol. Let me refactor.

Comment (sshanzel) [reply packages/shared/src/components/auth/RegistrationForm.tsx:35]: The `RegistrationFormValues` is the object that we need to pass Kratos, the `EmailRegistrationForm` is just for the scope of this component.

Though I could probably just be overly typing the objects, I think we can add it to the original interface.

**Review by rebelchris — APPROVED** [review]
Love it, nice refactor on the types 🙏

**Pull request merged**